image_output_path: '/home/aydin/data/IMG'
csv_dir: '/home/aydin/data/'
csv_name: 'driving_log.csv'
bagfile_name: 'data2' 
camera_topic: '/zed/zed_node/left_raw/image_raw_color'
CanBus_topic: '/joy_modified'
add_or_new: 'new'
published_topic: '/joy_modified'